charts/jaeger/values.yaml

```diff
@@ -2,6 +2,61 @@
 # This is a YAML-formatted file.
 # Jaeger values are grouped by component. Cassandra values override subchart values
 
+# Will use for user provided config
+userconfig:
+
+config:
```

Comment:
can we now remove this section?

Reply:
I am keep it for es i will remove it if I don't need it later